packages/storage/src/platform/node/connection.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ErrorCode, XhrIo } from '../../implementation/xhrio';
```

Comment:
Should we rename `XhrIo` to something generic? e.g. just `Io`. And rename`XMLHTTPRequestXhrIo` to just `XhrIo`?